_episodes/02-REPL.md

```diff
@@ -65,12 +66,13 @@ Great! This way she can easily look up the names she needs.
 She gets back to normal mode by pressing backspace.
 
 Another useful mode is the shell mode that can be entered by pressing <kbd>;</kbd>.
-It can be used to use commands of the underlying shell, but don't confuse it with an actual shell: Special shell syntax like piping won't work.
+The prompt has now changed to shell.
+It can be used to ~~use~~ issue commands of the underlying shell, but don't confuse it with an actual shell: Special shell syntax like piping won't work.
```

Comment:
```suggestion
It can be used to issue commands of the underlying shell, but don't confuse it with an actual shell: Special shell syntax like piping won't work.
```